portal/templates/instance_profile.html

```diff
@@ -92,6 +92,16 @@ <h1>Detailed services information currently not available</h1>
           <div role="tabpanel" class="tab-pane fade" id="logs">
             <br>
             <!-- <h3>Logs</h3> -->
+            <div id="logs-display">
+              <input type="number" id="nlines" name="nlines" value="20">
+              <button class="btn btn-primary btn-sm btn-box-shadow" v-on:click="getLogs">Submit</button>
+              <br>
+              <pre class="app-logs">
+                {[{logs}]}
```

Comment:
for testing, I just display the logs without the if conditions. the original logs display code block is kept at line 105 for reference